cottoncandy/interfaces.py

```diff
@@ -42,6 +42,11 @@
                     GzipInputStream,
                     DEFAULT_ACL, MPU_CHUNKSIZE, MPU_THRESHOLD, DASK_CHUNKSIZE, MB, SEPARATOR,
                     MAGIC_CHECK)
+from .options import config
+
+DO_COMPRESSION = config.get('compression', 'do_compression').lower() in ('true', 't', 'y','yes')
```

Comment:
you can use `cottoncandy.utils.string2bool` to convert strings to booleans:
https://github.com/gallantlab/cottoncandy/blob/master/cottoncandy/utils.py#L67